packages/calypso-build/webpack/util.js

```diff
@@ -67,4 +67,50 @@ function IncrementalProgressPlugin() {
 	return new webpack.ProgressPlugin( createProgressHandler() );
 }
 
-module.exports = { cssNameFromFilename, IncrementalProgressPlugin };
+const nodeModulesToTranspile = [
+	// general form is <package-name>/.
+	// The trailing slash makes sure we're not matching these as prefixes
+	// In some cases we do want prefix style matching (lodash. for lodash.assign)
+	'@github/webauthn-json/',
+	'acorn-jsx/',
+	'd3-array/',
+	'd3-scale/',
+	'debug/',
+	'escape-string-regexp/',
+	'filesize/',
+	'prismjs/',
+	'react-spring/',
+	'regenerate-unicode-properties/',
+	'regexpu-core/',
+	'striptags',
```

Comment:
oop, this one should have a trailing `/`

Reply:
If we go the semver route, and start exposing API for expanding the whitelist, then we should get rid of the trailing slashes and patch the matching algorithm instead.